### ISSUE TITLE
False positive PLE1206 when using star expansion in logging call

### ISSUE THREAD

**Issue by adamtheturtle** [charliermarsh/ruff#3242]
* ruff 0.0.252

```python
"""Demonstration of PLE1206 error."""

import logging

items = "foo", "bar"
logging.error("Example log %s, %s", *items)
```

```toml
[tool.ruff]
select = ["ALL"]
```

The code works, logging:

```
ERROR:root:Example log foo, bar
```

`ruff` shows:

```
example.py:6:1: PLE1206 Not enough arguments for `logging` format string
```
